business_requirement/models/business.py

```diff
@@ -453,3 +468,14 @@ class Project(models.Model):
     @api.depends('br_ids')
     def _br_count(self):
         self.br_count = len(self.br_ids)
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    """ Link the task and the business requirement """
+    business_requirement_id = fields.Many2one(
```

Comment:
@victormartinelicocorp  When we use the wizard to create the tasks, this field should be automatically assigned.